app/src/org/commcare/android/view/TabbedDetailView.java

```diff
@@ -102,54 +101,7 @@ public void onPageScrollStateChanged(int arg0) { }
     /*
      * Populate view with content from given Detail.
      */
-    public void setDetail(Detail detail) {
-        Detail[] details = detail.getDetails();
-
-        LinearLayout.LayoutParams pagerLayout = new LinearLayout.LayoutParams(LinearLayout.LayoutParams.MATCH_PARENT, LinearLayout.LayoutParams.WRAP_CONTENT);

```

Comment:
Just want to verify that you meant to remove the bulk of this method in your latest code review commit.